scrape_repo_information.py

```diff
@@ -0,0 +1,82 @@
+import urllib.request
+import pdb
+import xml.etree.ElementTree as xml
+import argparse, urllib.request
+import json
+import collections
+import time
+import urllib
+import os
+"""
+Script to scrape text from GitHub and into JSON format for all language pairs in Apertium.
+Stem counter by sushain
+"""
+
+CLIENT_ID = os.environ['GITHUB_CLIENT_ID']
```

Comment:
What happens if these are not set? If they are not set, can we just make the script print a warning that it will go slower and then make it go slower? Also, in the warning link to the GitHub docs on how to get these.